pkg/acsengine/params.go

```diff
@@ -231,13 +231,19 @@ func getParameters(cs *api.ContainerService, generatorCode string, acsengineVers
 
 		if properties.OrchestratorProfile.IsKubernetes() || properties.OrchestratorProfile.IsOpenShift() {
 			k8sVersion := properties.OrchestratorProfile.OrchestratorVersion
+
+			// Kubernetes packages as zip file as created by scripts/build-windows-k8s.sh
+			// will be removed in future release as if gets phased out (https://github.com/Azure/acs-engine/issues/3851)
 			kubeBinariesSASURL := properties.OrchestratorProfile.KubernetesConfig.CustomWindowsPackageURL
 			if kubeBinariesSASURL == "" {
 				kubeBinariesSASURL = cloudSpecConfig.KubernetesSpecConfig.KubeBinariesSASURLBase + api.K8sComponentsByVersionMap[k8sVersion]["windowszip"]
 			}
-
 			addValue(parametersMap, "kubeBinariesSASURL", kubeBinariesSASURL)
-			addValue(parametersMap, "windowsPackageSASURLBase", cloudSpecConfig.KubernetesSpecConfig.WindowsPackageSASURLBase)
+
+			// Kubernetes node binaries as packaged by upstream kubernetes
+			// example at https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.11.md#node-binaries-1
+			addValue(parametersMap, "windowsKubeBinariesURL", properties.OrchestratorProfile.KubernetesConfig.WindowsNodeBinariesURL)
```

Comment:
can we write a unit test to assert the presence of this key and its value. We have written similar uni tests in `params_k8s_test.go`

Reply:
The tests in `params_k8s_test.go` call to `params_k8s.go` by calling `assignKubernetesParameters()`. This is is in `params.go` under `getParameters()` which doesn't seem to have any unit tests.  

Should I try creating a test function like `TestAssignKubernetesParameters` in `params_k8s_test.go`? Or should i try to mimic the `TestKubernetesParamsAddons` but for the `kubernetesConfig` object?

Reply:
@jsturtevant Since this is a k8s-specific param. We should be populating this in `param_k8s.go`.

Reply:
In this case there are some kubernetes config settings that are where already set in the `params.go`  and since it directly correlated to the parameter above I would like to keep it next to it (`kubeBinariesSASURL`).  

We could move all these k8 specific params to the `params_k8s.go` but that should probably be done in a separate PR.

Reply:
Yes. I just realized. Sure. That can be done in a separate PR. As for your previous question, We can go with the former

Reply:
added the tests for the `params.go` file in a similar fashion as `params_k8s_test.go`